plugins/cost-insights-common/api-report.md

```diff
@@ -0,0 +1,185 @@
+## API Report File for "@backstage/plugin-cost-insights-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+// Warning: (ae-missing-release-tag) "ChangeStatistic" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
```

Comment:
Should, I be adding a `@beta` release tag to all the exported types in the new package?